console/src/schematic/services/ontology.tsx

```diff
@@ -117,6 +120,33 @@ const useRangeSnapshot = (): ((props: Ontology.TreeContextMenuProps) => void) =>
     },
   }).mutate;
 
+const useDownload = (): ((props: Ontology.TreeContextMenuProps) => void) =>
+  useMutation<void, Error, Ontology.TreeContextMenuProps, Tree.Node[]>({
+    mutationFn: async ({ client, selection: { resources }, store }) => {
+      let state = select(store.getState(), resources[0].id.key);
+      if (state == null) {
+        const schematic = await client.workspaces.schematic.retrieve(
+          resources[0].id.key,
+        );
+        state = schematic.data as unknown as State;
+        state.snapshot = schematic.snapshot;
```

Comment:
Should we preserve the snapshot field when exporting schematics? I don't think so.